demux/utils/samplesheet.py

```diff
@@ -282,7 +285,7 @@ def _validate_sample_name(samplesheet):
                         i + 2,
                     )
 
-        rs = _validate_unique_index(self.samplesheet)
+        rs = _validate_uniq_index(self.samplesheet)
```

Comment:
```suggestion
        rs = _validate_unique_index(self.samplesheet)
```

Maybe it's best to revert some stuff, this was all put in by me, very much intentionally.